Sources/Base/ASTContext.swift

```diff
@@ -762,7 +762,10 @@ public class ASTContext {
     }
   }
   
-  func canCoerce(_ type: DataType, to other: DataType) -> Bool {
+  func canCoerce(_ _type: DataType, to _other: DataType) -> Bool {
```

Comment:
since the raw args are never used, why do they need to have the underscored names?